src/main/WikiVectors.java

```diff
@@ -131,9 +131,13 @@ void computeByteVectors(String lineDocFile, OutputStream out) throws IOException
     try (Reader r = Channels.newReader(FileChannel.open(Paths.get(lineDocFile)), dec, -1);
          BufferedReader in = new BufferedReader(r)) {
       String lineDoc;
+      byte[] bvec = new byte[dict.dimension];
       while ((lineDoc = in.readLine()) != null) {
-        byte[] vec = (byte[]) dict.computeTextVector(lineDoc);
-        out.write(vec);
+        float[] vec = dict.computeTextVector(lineDoc);
+        for (int i = 0; i < vec.length; i++) {
+          bvec[i] = (byte) vec[i];
+        }
+        out.write(bvec);
```

Comment:
Oooh `byte` -> `float` vectors ... will this change the nightly vectors indexing test too?  We should add an annotation if so.

Reply:
I think this was a bug and this code-path was not being exercised because if it were it would have failed with ClassCastException?